itsy.html

```diff
@@ -66,5 +66,65 @@
+        <blockquote>
+          <p >"There are always two people in every picture: the photographer and the viewer."</p>
+          <p> - Ansel Adams</p>
+        </blockquote>
+        <p class="post-content__text lato">Nullam ac odio non ligula semper auctor. Fusce semper risus eu magna placerat pulvinar. Nullam ac odio non ligula semper auctor. Aenean at dui dui, non scelerisque nisi. Morbi ullamcorper dapibus nisl, ullamcorper fringilla eros pulvinar et.</p>
+        <button class="amatic"> read more</button>
+        <p class="post-content__tag">saved: blog, fitness</p>
+      </div>
+    </div>
+    <div class="post-item amatic">
+      <div class="post-content">
+        <p class="post-content__title">what a cute template</p>
+        <p class="post-content__description">november 6, 2014 <i class="fas fa-circle "></i> 0 COMMENTS <i class="fas fa-circle"></i> BY CLARE SMITH</p>
+        <button class="amatic" id="post-content__btn"> visit website</button>
+        <p class="post-content__tag">saved: blog, fitness</p>
+    </div>
+  </div>
+    <div class="post-item amatic">
+      <div class="post-item__preview">
+        <img src="img/Camper-1.jpg">
+      </div>
+      <div class="post-content">
+        <p class="post-content__title">camper van fun</p>
+        <p class="post-content__description">october 6, 2014 <i class="fas fa-circle "></i> 2 COMMENTS <i class="fas fa-circle"></i> BY CLARE SMITH</p>
+        <p class="post-content__text lato">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce at risus at lacus laoreet mollis sed id elit. Integer bibendum lobortis velit, eleifend commodo dui facilisis nec. Aliquam mi sapien, ultrices a ultrices non, sodales ut diam. Fusce semper risus eu magna placerat pulvinar. Nullam ac odio non ligula semper auctor. Fusce semper risus eu magna placerat pulvinar.</p>
+        <button class="amatic"> read more</button>
+        <p class="post-content__tag">saved: blog, fitness</p>
+      </div>
+    </div>
+    <div class="post-item amatic">
+      <div class="post-content">
+        <p class="post-content__title">a lovely quote</p>
+        <p class="post-content__description">october 1, 2014 <i class="fas fa-circle "></i> 0 COMMENTS <i class="fas fa-circle"></i> BY CLARE SMITH</p>
+        <blockquote>
+          <p >"There are always two people in every picture: the photographer and the viewer."</p>
+          <p> - Ansel Adams</p>
+        </blockquote>
+        <p class="post-content__tag">saved: blog, fitness</p>
+      </div>
+    </div>
+    <div class="post-item amatic">
+      <iframe class="post-iframe" src="https://player.vimeo.com/video/33091687?portrait=0" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
```

Comment:
shouldn't be a space here
<img width="1104" alt="image" src="https://user-images.githubusercontent.com/8299599/105633827-95a00e80-5e63-11eb-83dd-322701aa7261.png">